app-backend/common/src/main/scala/datamodel/export/ExportDefinition.scala

```diff
@@ -0,0 +1,90 @@
+package com.rasterfoundry.common.datamodel.export
+
+import com.azavea.maml.ast._
```

Comment:
Some unused imports here and in `backsplash-export` subproject -- would be good to run` scalafix RemoveUnused` in those two subprojects before merging

Reply:
scalafix is incredible.

Reply:
The mask you're using appears to be WebMercator; I believe the reprojection changes in db should circumvent this: https://github.com/raster-foundry/raster-foundry/pull/4589/files#diff-efc1e81bc82715fb8279ee27b0c4374dR169